examples/ex71.hpp

```diff
@@ -0,0 +1,587 @@
+// shared implementation ex71p/ex71 for the AD integrands and
+// the handconded integrators
+
+
+#ifndef EXAMPLE71_H
+#define EXAMPLE71_H
+
+#include "mfem.hpp"
+#include <memory>
+#include <iostream>
+#include <fstream>
+
+
+namespace mfem
+{
+
+
+template<typename DType, typename MVType>
+class MyQFunctorJ
+{
+public:
+   DType operator()(const mfem::Vector& vparam, MVType& uu)
```

Comment:
The requirements for how to define these functors should be documented.

Reply:
Documented!